setup.py

```diff
@@ -40,7 +44,8 @@
     url='http://github.com/ojii/django-classy-tags',
```

Comment:
Please add the django-cms repo name, it still points to the old repo link of ojji

Reply:
Good spot. I thought I did that with the release last week